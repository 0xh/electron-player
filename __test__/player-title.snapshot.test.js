/**
 * @author arman
 * @since 8/23/17
 *
 */

import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../app/components/dumb/player-title.component';

test('Title component renders the title correctly', () => {
  const rendered = renderer.create(
    <Title text="Electron Player" />
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});
