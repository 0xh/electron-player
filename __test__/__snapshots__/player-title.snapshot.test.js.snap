// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders the title correctly 1`] = `
<h1
  className="title deep-shadow"
>
  Electron Player
</h1>
`;
